unittests/test_gdicmn.py

```diff
@@ -15,7 +18,10 @@ def test_xy_ctor(self):
 
     def test_RealPoint_ctor(self):
         p = wx.Point(wx.RealPoint(1.2, 2.9))
-        self.assertTrue(p == (1,2))
+        if viTuple < (3, 1, 2):
```

Comment:
Why support wx < 3.1.2?

Reply:
Sou you can have identical code in the 4.0 and 4.1 branches

Reply:
I don't think Robin is really maintaining the 4.0.x branch any more.  Certainly test fixes wouldn't be likely to be backported.

Reply:
The master branch is for 4.1 and beyond, it would be undesirable to clutter it up for things that are specific to a prior release series like 4.0. Changes and PRs that are 4.0.x specific would need to be applied to the wxPy-4.0.x branch, although as @swt2c said, it's unlikely that there will be another release in that series. 

Also, it would have been more appropriate to use `wx.VERSION` anyway, instead of the wxWidgets version number.

Reply:
> Also, it would have been more appropriate to use `wx.VERSION` anyway, instead of the wxWidgets version number.

Definitely not - wx.VERSION >= 4.1 refers to _any_ wxWidgets version >= 3.1, but this changed between 3.1.1 and 3.1.2.

Reply:
Oh, I see. I didn't look closely enough to see that's where you were making the demarcation. Regardless, `master` should be forward looking. If there's ever a need to make a build from wxWidgets 3.1.1 then we'll just checkout the wxPython revision that has a submodule link to the desired revision of wxWidgets, and end up with the older version of the test code as well.